docs/theming.md

```diff
@@ -4,11 +4,12 @@ Theming Element
 Themes are a very basic way of providing simple alternative look & feels to the
 Element app via CSS & custom imagery.
 
-They are *NOT* co be confused with 'skins', which describe apps which sit on top
+They are *NOT* to be confused with 'skins', which describe apps which sit on top
 of matrix-react-sdk - e.g. in theory Element itself is a react-sdk skin.
 As of Jan 2017, skins are not fully supported; Element is the only available skin.
 
-To define a theme for Element:
+To define a CSS based theme for Element:
+=========================================
```

Comment:
I think I would change the very first line to be "CSS Based Themes:" rather than changing this line.